src/semgrep_agent/formatter.py

```diff
@@ -68,9 +84,69 @@ def _print_error_message(violation: Finding) -> str:
     return f"{BOLD}{rule}{END}"
 
 
-def dump(findings: Set[Finding]) -> None:
+OSC_8 = "\x1b]8;;"
+BEL = "\x07"
+
+
+def is_child_process_of(pattern: Pattern) -> bool:
+    """
+    Returns true iff this process is a child process of a process whose name matches pattern
```

Comment:
```suggestion
    Returns true iff this process is a child process of a process whose name contains pattern
```

(matches implies `re.match`-like behavior)